assets/data/formation_pools.ron

```diff
@@ -1,6 +1,36 @@
 (	
 	formation_pools: {
 		"easy": [
+			(
+				formation_spawnables: [
```

Comment:
This formation seems a little too challenging for the easy pool. Maybe reduce the number of enemies and/or increase the period?